trigger/virtcontainers/vc.go

```diff
@@ -57,6 +59,58 @@ func getSocketPath() (string, error) {
 	return socketURI, nil
 }
 
+func waitForDirectory(dir string) error {
+	type result struct {
+		message string
+		err     error
+	}
+
+	syncCh := make(chan result)
```

Comment:
It doesn't appear that `result.message` is really used - could this be simplified to just `make(chan error)`?